web/site/nuxt.config.ts

```diff
@@ -46,7 +53,17 @@ export default defineNuxtConfig({
       'en-CA'
       // 'fr-CA'
     ],
-    contentHead: false
+    contentHead: false,
+    highlight: {
+      theme: {
+        default: 'github-light',
+        dark: 'github-dark'
+      }
+    },
+    ignores: [
```

Comment:
ignore web component and tos for now. web component will need to be reworked to show previews